language/bytecode-verifier/src/verifier.rs

```diff
@@ -234,105 +231,6 @@ pub fn verify_script_dependency_map(
     verify_dependencies(&fake_module, dependency_map)
 }
 
-fn verify_native_functions(module_view: &ModuleView<VerifiedModule>) -> VMResult<()> {
```

Comment:
part of this is gone forever and the part that matters and was used at runtime is in the loader

Reply:
We should still verify that the native function exists, no?

Reply:
not in the verifier, how would the verifier know? the VM is the only one that knows, no?

Reply:
I understand how things are laid out now